docs/webhooks/index.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en">
 
 <head>
-    <link rel="icon" type="image/png" href="https://codesignal.s3.amazonaws.com/uploads/1553569752846/favicon.png"/>
+    <link rel="icon" type="image/png" href="https://codesignal.com/wp-content/uploads/2020/11/CodeSignal-Logo_Circle_Blue.svg" sizes="32x32"/>
+                                           <!-- https://codesignal.s3.amazonaws.com/uploads/1553569752846/favicon.png -->
```

Comment:
No problem updating the favicon (thank you!), so let's go ahead and remove the old icon instead of just commenting it out. It's always preserved in Git history if we need to find it again for some reason.

```suggestion
    <link rel="icon" type="image/png" href="https://codesignal.com/wp-content/uploads/2020/11/CodeSignal-Logo_Circle_Blue.svg" sizes="32x32"/>
```